docs/snippets/web-components/simple-page-implementation.ts.mdx

```diff
@@ -0,0 +1,39 @@
+```ts
```

Comment:
@kasperpeulen — Does this implementation look right to you?

Used here: https://storybook.js.org/docs/7.0/web-components/writing-stories/build-pages-with-storybook#args-composition-for-presentational-screens